docs/tutorial.md

```diff
@@ -106,6 +106,16 @@ model Dog extends Animal {}
 
 ```
 
+```CSharp
```

Comment:
This feels out of place to me. It could be a different C#-specific doc that shows equivalences of Cadl constructs to C#, and we could show lots of other equivalences, and give the same treatment for other languages.